docs/reference/generated/kubelet.md

```diff
@@ -124,7 +124,7 @@ TaintNodesByCondition=true|false (ALPHA - default=false)
       --file-check-frequency duration                            Duration between checking config files for new data (default 20s)
       --google-json-key string                                   The Google Cloud Platform Service Account JSON Key to use for authentication.
       --hairpin-mode string                                      How should the kubelet setup hairpin NAT. This allows endpoints of a Service to loadbalance back to themselves if they should try to access their own Service. Valid values are "promiscuous-bridge", "hairpin-veth" and "none". (default "promiscuous-bridge")
-      --healthz-bind-address ip                                  The IP address for the healthz server to serve on. (set to 0.0.0.0 for all interfaces) (default 127.0.0.1)
+      --healthz-bind-address ip                                  The IP address for the healthz server to serve on. (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 127.0.0.1)
```

Comment:
should be fixed upstream here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L430